src/services/passwordGeneration.service.ts

```diff
@@ -473,4 +506,54 @@ export class PasswordGenerationService implements PasswordGenerationServiceAbstr
             [array[i], array[j]] = [array[j], array[i]];
         }
     }
+
+    private sanitizePasswordLength(options: any, forGeneration: boolean) {
+        let minUppercaseCalc: number = 0;
+        let minLowercaseCalc: number = 0;
```

Comment:
You dont need to specify type here since you are direct assigning it as a number already.

```suggestion
        let minUppercaseCalc = 0;
        let minLowercaseCalc = 0;
```

Reply:
Updated.